### PR TITLE
[jsDelivr] badges

### PR DESCRIPTION
**Pull request by Prouser123**
Requested in #2862.

Currently NPM + GitHub hit badges.

I don't personally feel that a versioning badge is necessary (jsDelivr being a CDN) but feel free to prove me wrong and I'll happily add it in.

### CODE REVIEW

File: services/jsdelivr/jsdelivr-hits-github.service.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    const { total } = await this.fetch({ period, user, repo })
+    return this.constructor.render({ period, hits: total })
+  }
+
+  async fetch({ period, user, repo }) {
+    const url = `https://data.jsdelivr.com/v1/package/gh/${user}/${repo}/stats/date/${
+      periodMap[period]
+    }`
+
+    return this._requestJson({
```
Comment (MartinKolarik) [inline services/jsdelivr/jsdelivr-hits-github.service.js]: Please add a `User-Agent` header [as recommended](https://github.com/jsdelivr/data.jsdelivr.com#let-us-know-how-you-use-this-api).

File: services/jsdelivr/jsdelivr-base.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  hw: 'week',
+  hm: 'month',
+  hy: 'year',
+}
+
+class BaseJsDelivrService extends BaseJsonService {
+  static render({ period, hits }) {
+    return {
+      message: `${metric(hits)} hits/${periodMap[period]}`,
+      //color: downloadCount(hits),
```
Comment (chris48s) [inline services/jsdelivr/jsdelivr-base.js]: Can we get rid of this line if we're not using it.

File: services/jsdelivr/jsdelivr-base.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    }
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'jsDelivr',
+      color: 'orange',
```
Comment (chris48s) [inline services/jsdelivr/jsdelivr-base.js]: If we want to just make everything one colour, can we go with blue (which we use for "information" as opposed to good/bad/etc).

File: services/jsdelivr/jsdelivr-base.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  static render({ period, hits }) {
+    return {
+      message: `${metric(hits)} hits/${periodMap[period]}`,
+      //color: downloadCount(hits),
+    }
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'jsDelivr',
```
Comment (chris48s) [inline services/jsdelivr/jsdelivr-base.js]: The left-hand side of the badge should be all lowercase: https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-guidelines (even if the service usually styles their name in mixed-case)

File: services/jsdelivr/jsdelivr-base.js (hunk 3 above)
Comment (Prouser123) [reply services/jsdelivr/jsdelivr-base.js]: I used orange as that was the color scheme used on their website and their badges.

**Pull request merged**